Fix logging and improve DX (#3047)
open-condo-software/condo: packages/featureflags/README.md

If you need to hide some functionality or give access to it to a certain group of users, 
then you need to use feature flags.

> To use it you need to add `FeaturesMiddleware` to you `index.js`

Reviewer: Specify, where to find it :)

Author: This package has only tree file and one of them is FeaturesMiddleware %) 
But ok, I'll explain it %))

Reviewer: ahahah, my bad :) 

I was looking for GitHub diff only and it showed, that it has only has `...manager.js` file, that's I was struggling to find a  middleware 😄

> To use it you need to add `FeaturesMiddleware` to you `index.js`

Reviewer 2: ```suggestion
Add `FeaturesMiddleware` to your `index.js`
```

***

## Use it locally:

> 1) You can define a feature statically (only for local development)! Add follow local variable:

Reviewer 2: ```suggestion
1) You can define a feature statically for local development only via adding following variables to `.env`:
```

`FEATURE_TOGGLE_CONFIG='{"url":null,"apiKey":null,"static": {"sms-after-ticket-creation":{"defaultValue":false,"rules":[{"condition":{"organization":{"$in":[]}},"force":true}]},"refetch-tickets-in-control-room":{"defaultValue":false,"rules":[{"force":true}]},"ticket-import":{"defaultValue":false,"rules":[{"condition":{"isSupport":true},"force":true}]},"send-billing-receipts-notifications-task":{"defaultValue":true},"max-count-completed-ticket-to-close-for-organization-task":{"defaultValue":100}}'`

> 2) You can setUp a GrowBook service locally: 

Reviewer 2: ```suggestion
2) You can setup a GrowBook service locally: 
```

   2.1) [Raise the GrowthBook locally](https://docs.growthbook.io/self-host) 

   2.2) Get API key from Settings -> API Keys in GrowthBook interface and pass it and API URL to `FEATURE_TOGGLE_CONFIG` in `.env`.
        (Example: `FEATURE_TOGGLE_CONFIG='{"url": "http://localhost:3100/api/features", "apiKey": "key_prod_1234abcd"}'`)

>    2.3) Create a new feature toggle in GrowthBook interface

Reviewer 2: ```suggestion
   2.3) Create a new feature toggle in GrowthBook dashboard
```
